modules/quantcastBidAdapter.js

```diff
@@ -112,12 +102,21 @@ export const spec = {
     const page = utils.deepAccess(bidderRequest, 'refererInfo.canonicalUrl') || config.getConfig('pageUrl') || utils.deepAccess(window, 'location.href');
     const domain = getDomain(page);
 
-    const bidRequestsList = bids.map(bid => {
+    let bidRequestsList = [];
+
+    bids.forEach(bid => {
       let imp;
-      const videoContext = utils.deepAccess(bid, 'mediaTypes.video.context');
-      if (videoContext === 'instream') {
-        imp = makeVideoImp(bid);
+      if (bid.mediaTypes) {
+        if (bid.mediaTypes.video && bid.mediaTypes.video.context === 'instream') {
+          imp = makeVideoImp(bid);
+        } else if (bid.mediaTypes.banner) {
+          imp = makeBannerImp(bid);
+        } else {
+          // Unsupported mediaType
```

Comment:
Do you want to throw a warning/info message message here to note that an unsupported bid type was ignored by the adapter (if only for future diagnostic information)?

Reply:
:+1: done